x/evm/artela/api/evm_api.go

```diff
@@ -74,7 +73,7 @@ func (e evmHostApi) StaticCall(ctx *coretypes.RunnerContext, request *coretypes.
 		ProposerAddress: nil,
 		ChainId:         chainID.Int64(),
 	}
-	call, ethErr := e.ethCall(sdkCtx.Context(), ethRequest)
+	call, ethErr := e.ethCall(aspectCtx, ethRequest)
```

Comment:
The ethCall function requires passing a Cosmos context. Please review the implementation of EthCall and ensure that the test code covers this line.